website/docs/components/form/super-select/partials/code/how-to-use.md

```diff
@@ -0,0 +1,85 @@
+## How to use this component
+
+SuperSelect is a custom select-like component aiming to overcome some limitations of the HTML `<select>` element.
+It is based on the popular Ember [PowerSelect](https://ember-power-select.com/) add-on.
+
+We provide two variants, `SuperSelect::Single` and `SuperSelect::Multiple`.
+
+**There are two ways to use the Super Select variants:**
```

Comment:
[Suggestion] Maybe it makes sense to consolidate this with the previous sentence? Like:
```suggestion
We provide two variants, `SuperSelect::Single` and `SuperSelect::Multiple` that be can used in one of two ways:
```

Reply:
In code, they are two separate components but they are like variants since they're almost completely the same in terms of API. Do you think it's best (clearer?) to not call them "variants" though?

Reply:
With this description, "variant" does seem a bit like a loaded term because designers might then expect that they can bring in a single Figma component and swap between the single/multi variants as properties of the same component which is not really the case. Maybe it's as simple as saying something like:

"We provide two components that share similar APIs, `SuperSelect::Single` and `SuperSelect::Multiple`…"

That's not totally perfect, but I think it makes sense to differentiate them if they aren't _technically_ variants.

Reply:
Thanks @jorytindall That's valid. I will go through code-related docs to remove/replace the term "variant".